src/ts/components/Match.tsx

```diff
@@ -8,13 +8,19 @@ interface IMatchProps<TMatch extends IMatch> {
   match: TMatch;
 }
 
+const getFeedbackLink = (feedbackInfo: any) => {
+  const data = encodeURIComponent(JSON.stringify(feedbackInfo, undefined, 2))
+  return "https://docs.google.com/forms/d/e/1FAIpQLSfMOgvJtCchnW0_2zB7Afz_WtYJ5lnPqQI-dgFZ-p0B4h6uKw/viewform?usp=pp_url&entry.110962249=" + data
+}
+
 class Match<TMatch extends IMatch> extends Component<IMatchProps<TMatch>> {
   public ref: HTMLDivElement | null = null;
   public render({
     match: { matchId, category, message, suggestions, replacement },
     applySuggestions
   }: IMatchProps<TMatch>) {
     const suggestionsToRender = replacement ? [replacement] : suggestions || [];
+    const feedbackInfo = { matchId, category, message, suggestions, replacement };
```

Comment:
I don't think this includes the original match text. Which could be helpful in determining if a match has been mistakenly made.

Reply:
Yes - that's too hard for me for now.  I'm going to do that as stage 2.